verify.go

```diff
@@ -17,53 +17,51 @@ import (
 // IDTokenVerifier provides verification for ID Tokens.
 type IDTokenVerifier struct {
 	keySet *remoteKeySet
-	config *verificationConfig
+	config *Config
+	issuer string
 }
 
-// verificationConfig is the unexported configuration for an IDTokenVerifier.
-//
-// Users interact with this struct using a VerificationOption.
-type verificationConfig struct {
-	issuer string
-	// If provided, this value must be in the ID Token audiences.
-	audience string
-	// If not nil, check the expiry of the id token.
-	checkExpiry func() time.Time
+// Config is the configuration for an IDTokenVerifier.
+type Config struct {
+	// If provided, ClientID must be in the ID Token audiences.
```

Comment:
This comment seems to imply that it's okay to not provide this value. We might want to change it to say something like.

```
// The ID of the client that started the login flow, which is expected to be in the ID Token's audiences.
//
// If not provided, users must explicitly set SkipClientIDCheck.
```